Prox/Prox/AppDelegate.swift

```diff
@@ -75,7 +75,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(_ application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
```

Comment:
How do we activate this? Does it happen automatically with the `*.plist` category?